kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.styled.ts

```diff
@@ -3,41 +3,54 @@ import { Colors } from 'theme/theme';
 
 import { TableHeaderCellProps } from './TableHeaderCell';
 
-export const TableHeaderCell = styled.th<TableHeaderCellProps>`
-  padding: 4px 0 4px 24px !important;
-  border-bottom-width: 1px !important;
-  vertical-align: middle !important;
+interface TitleProps {
+  isSortable?: boolean;
+  isCurrentSort?: boolean;
+}
+export const Title = styled.span<TitleProps>`
+  font-family: Inter, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 16px;
+  letter-spacing: 0em;
+  text-align: left;
+  background: ${(props) => props.theme.thStyles.backgroundColor.normal};
+  color: ${(props) => props.theme.thStyles.color.normal};
 
-  &.is-clickable {
-    cursor: pointer !important;
-    pointer-events: all !important;
-  }
+  ${(props) =>
+    props.isSortable &&
+    `
+    cursor: pointer;
 
-  &.has-text-link-dark span {
-    color: ${Colors.brand[50]} !important;
-  }
+    &:hover {
+      color: ${Colors.brand[50]};
+    }
+  `}
 
-  span {
-    font-family: Inter, sans-serif;
-    font-size: 12px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 16px;
-    letter-spacing: 0em;
-    text-align: left;
-    background: ${(props) => props.theme.thStyles.backgroundColor.normal};
-    color: ${(props) => props.theme.thStyles.color.normal};
+  ${(props) =>
+    props.isCurrentSort &&
+    `
+    color: ${Colors.brand[50]};
+  `}
+`;
 
-    &.preview {
-      margin-left: 8px;
-      font-size: 14px;
-      color: ${(props) => props.theme.thStyles.previewColor.normal};
-      cursor: pointer;
-    }
+export const Preview = styled.span`
+  margin-left: 8px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 16px;
+  letter-spacing: 0em;
+  text-align: left;
+  background: ${(props) => props.theme.thStyles.backgroundColor.normal};
+  font-size: 14px;
+  color: ${(props) => props.theme.thStyles.previewColor.normal};
+  cursor: pointer;
+`;
 
-    &.is-clickable {
-      cursor: pointer !important;
-      pointer-events: all !important;
-    }
-  }
+export const TableHeaderCell = styled.th<TableHeaderCellProps>`
+  padding: 4px 0 4px 24px !important;
+  border-bottom-width: 1px !important;
+  vertical-align: middle !important;
```

Comment:
I'm not sure we really need all these `!important`. Could you double check?

Reply:
Removed `!important`
Looks the same